src/core/server/saved_objects/migrationsv2/migrations_state_action_machine.ts

```diff
@@ -49,14 +49,15 @@ const logStateTransition = (
   tookMs: number
 ) => {
   if (newState.logs.length > oldState.logs.length) {
+    const doLog = (level: MigrationLogLevel, message: string) => {
```

Comment:
nit: do we really need to create this function? it might be inlined in the `forEach` body.